test/new-e2e/tests/otel/otel-agent/complete_test.go

```diff
@@ -0,0 +1,59 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package otelagent contains e2e otel agent tests
+package otelagent
+
+import (
+	_ "embed"
```

Comment:
asking out of ignorance, why is this added as "_"?

Reply:
We're only using this package to embed yaml files into strings (`//go:embed config/complete.yml`), but aren't using any of its functions so it needs to be a blank import: https://pkg.go.dev/embed#hdr-Strings_and_Bytes.